compiler/bin-wasm_of_ocaml/compile.ml

```diff
@@ -270,15 +272,8 @@ let run
   List.iter builtin ~f:(fun t ->
       let filename = Builtins.File.name t in
       let runtimes = Linker.Fragment.parse_builtin t in
-      Linker.load_fragments
-        ~ignore_always_annotation:true
```

Comment:
To remove this argument, we need https://github.com/ocsigen/js_of_ocaml/pull/1674/